rental.rb

```diff
@@ -0,0 +1,7 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] please remember a rental belongs to a book and also belongs to a person. so Getters and Setters for @book and @person instance variables should be present

`attr_accessor :date, :book, :person`